lib/literal_helpers.ex

```diff
@@ -4,7 +4,13 @@ defmodule Expression.LiteralHelpers do
 
   def int do
     optional(string("-"))
-    |> concat(integer(min: 1))
+    |> times(
+      choice([
```

Comment:
what's `choice`?

Reply:
it means the parser *must* choose one of the two options, either we hit a integer, or we hit an underscore (which we then ignore).

The choice function is documented here https://hexdocs.pm/nimble_parsec/NimbleParsec.html#choice/2
The times function is documented here https://hexdocs.pm/nimble_parsec/NimbleParsec.html#times/3